changelog/fix-1152-wcpay-checkout-payment-methods-fields-visibility

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Improve visibility of checkout fields for WCPay payment options on a darker theme
```

Comment:
We can skip the changelog in this PR, as we'll be merging our work in a dedicated branch, where the single changelog entry will be introduced.

Reply:
I'm not sure from where those changes appeared 🤔 I think it's because I rebased on `develop` instead of `dev/pulsar-hackweek-ts`. Looking into it

Reply:
Indeed. I reverted the changes and it looks much better now. Thank you for changing the base branch